kartothek/io_components/metapartition.py

```diff
@@ -183,6 +183,29 @@ class MetaPartition(Iterable):
     about the parent dataset
     """
 
+    _METADATA_COLUMNS = (
```

Comment:
Can you please define this as one dict instead? having two tuples makes the whole thing hard to read

Reply:
Reason for constructing it this way was to make sure I can't mess with these constants at runtime.
Sure, the risk of that happening might be low and probably also in this specific use case.

I guess the reason why I did it this way is because I had a bug in the last weeks which was a bit nasty because it manipulated some class variable at runtime. And using an immutable data type felt like the right data type for "This is constant, please don't mess with it", which I don't have when I use a dict.